webcrypto/algorithm.go

```diff
@@ -178,6 +187,8 @@ func isRegisteredAlgorithm(algorithmName string, forOperation string) bool {
 		return isAesAlgorithm(algorithmName)
 	case OperationIdentifierSign, OperationIdentifierVerify:
 		return algorithmName == HMAC || algorithmName == ECDSA
+	case OperationIdentifierDeriveBits, OperationGetKeyLength:
```

Comment:
Shouldn't it be `OperationIdentifierDeriveKey` instead? 🙇‍♂️ (note that maybe `deriveKey` might rely on the bits derivation under the hood and I'm not just aware of it)

Reply:
Yeah step 2 in the [spec](https://www.w3.org/TR/WebCryptoAPI/#SubtleCrypto-method-deriveKey) for deriveKey is to deriveBits. What I essentially found is to fully implement deriveKey I think the issues with deriveBits are going to need fixed. So [this](https://github.com/grafana/xk6-webcrypto/issues/80) is probably going to be a side mission that needs completed before moving on to the main quest :smile: It looks like I also need a getKeyLength function and from what I can tell that is not implemented in any of the libraries that you suggested so I think that has to be written ourselves as well.

Reply:
Yeah that is what I found the step 2 in the spec [here](https://www.w3.org/TR/WebCryptoAPI/#SubtleCrypto-method-deriveKey) for deriveKey requires deriveBits. So I think the [issues](https://github.com/grafana/xk6-webcrypto/issues/80) for deriveBits need worked on first before I can continue so that is my current side mission 😄 I also realized that step 6 requires a "get key length" which was the util function I was referring to in my part 2 of the question from the other day. I think that is also going to be needed for the deriveBits issues so 2 birds 1 stone type of thing going on. Either way I am fairly sure that function will need to be our own implementation because I am not seeing it in any of the libraries that you mentioned.